docker/oso-saml-sso/README.adoc

```diff
@@ -0,0 +1,31 @@
+= simpleSAMLphp SSO application Docker container
+
+This docker container provides an HTTP endpoint and an SSH endpoint
+for single sign-on (SSO) to a SAML2 identity provider (IdP). The SSO
+can be used to authenticate to configured SAML2-enabled service
+providers (SPs).
+
+== Setup
+See https://github.com/openshift/openshift-ops/blob/pr/docker/README.adoc#setup[general setup].
```

Comment:
All the other docker containers in the container reference the same broken link. Do you know what this was supposed to point to? Otherwise, what is supposed to go in the setup section of this doc?